roles/dev_deploy/tasks/common.yml

```diff
@@ -61,29 +61,33 @@
 
 - debug: var=result.stdout_lines
 
-- name: Generate albs_jwt_token
+- name: Set albs_jwt when is not set via vars.yml
   tags: 
     - jwt_tokens
     - albs_jwt_token
-  shell: >
-    python3 -c "import jwt; import datetime; secret = '{{ albs_jwt_secret }}'; payload = {'sub': '1', 'aud': ['fastapi-users:auth'], 'exp': 1777628461}; token = jwt.encode(payload, secret, algorithm='HS256'); print(token)"
-  register: albs_jwt_result
-  when: albs_jwt_token is not defined or albs_jwt_token == ""
-
-- name: Set albs_jwt_token variable
-  tags:
-    - jwt_tokens
-    - albs_jwt_token
+  block:
+    - name: Generate ALBS JWT
+      shell: >
+        python3 -c "import jwt; secret = '{{ albs_jwt_secret }}'; payload = {'sub': '1', 'aud': ['fastapi-users:auth'], 'exp': 1777628461}; token = jwt.encode(payload, secret, algorithm='HS256'); print(token)"
```

Comment:
I think you can put the code into a file, copy it to a host and run it directly.